<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>475</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI Semilight</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>QR Scraper</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="titleVerticleLayout">
       <item>
        <widget class="QLabel" name="qrcodescraperLabel">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>YouTube                      QR Code Scraper</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="creatorLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>by Tom Opolski</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="displayVideosCheckBox">
         <property name="text">
          <string>Display videos being processed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="getWebsitesCheckBox">
         <property name="text">
          <string>Get Websites</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="getTcgCodesCheckBox">
         <property name="text">
          <string>Get PKMN TCG Codes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>293</y>
        <width>481</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="outputVerticalLayout">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Output List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="outputDataList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="urlVerticalLayout">
       <item>
        <widget class="QLabel" name="youtubeUrlLabel">
         <property name="text">
          <string>YouTube URL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userLinkEdit">
         <property name="text">
          <string>https://www.youtube.com/watch?v=oAcumtLhp58</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="searchLinkCheckBox">
         <property name="text">
          <string>Search link (vs. Video Link)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addVideoButton">
         <property name="text">
          <string>Add to Video Queue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>231</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="queueVerticalLayout">
       <item>
        <widget class="QLabel" name="videoQueueLabel">
         <property name="text">
          <string>Video Queue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="videoQueueList"/>
       </item>
       <item>
        <widget class="QPushButton" name="scanVideosButton">
         <property name="text">
          <string>Scan Videos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearVideosButton">
         <property name="text">
          <string>Clear Videos</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="aboutVerticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Created by: Tom Opolski</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Date: 12/30/19</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Website: https://github.com/opolskitom (I really need a website)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
